cypress/obj/loginPage.js

```diff
@@ -0,0 +1,88 @@
+class LoginPage 
+{
+    visit()
+    {
+        cy.visit('/');
+    }
+    getUserValidationIndicator()
+    {
+        return cy.get('.Select.not-valid');
+    }
+
+    getSubmitButton()
+    {
+        return cy.get('[type="submit"]');
+    }
+
+    getSpecificUserFromDropDown(userName)
+    {
+        return cy.get('[aria-label="' + userName + '"]');
+    }
+
+    openUpUserDropDown()
+    {
+        return cy.get('[id="loginForm.userId"]');
+    }
+
+    openUpRoleDropDown()
+    {
+        return cy.get('[id="loginForm.role"]');
+    }
+
+    getSpecificRoleFromDropDown(role)
+    {
+        return cy.get('[aria-label="' + role + '"]');
+    }
+
+    checkIfWeAreInSpecificURL(url)
+    {
+        return cy.url().should('include', url);
+    }
+
+    getPageTitle()
+    {
+        return cy.get('.page__title');
+    }
+    
+    getCalendar()
+    {
+        return cy.get('.calendar');
+    }
+
+    getTitleForm()
+    {
+        return cy.get('.tile.form');
+    }
+
+    checkIfInfoTitleIsSpecific(nameSurname)
```

Comment:
The name of this function could be more descriptive, e.g. `checkLoginName(nameSurname)`, so that we know from the function name that were talking about checking our name. Checking the "info title" might not seem very clear :)